index.html

```diff
@@ -4,32 +4,36 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Alexander Kaiser</title>
+    <link
+      href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1 id="top">Alexander Kaiser</h1>
+  <body class="m-4">
+    <h1 id="top" class="text-3xl font-bold mb-4">Alexander Kaiser</h1>
     <nav>
-      <ul>
+      <ul role="list">
         <li><a href="#contact">Contact</a></li>
         <li><a href="#biography">Biography</a></li>
       </ul>
     </nav>
     <main>
       <img src="images/alexander.jpg" alt="Alexander Kaiser" />
-      <h2>About me</h2>
+      <h2 id="aboutme" class="text-2xl font-bold">About me</h2>
       <p>
         My name is Alexander Kaiser. I am 24 yours old and live in Wippingen,
         Lower Saxony. At the moment I am participating in a web development
         bootcamp.
       </p>
-      <h3>Hobbies</h3>
+      <h3 class="text-xl font-bold">Hobbies</h3>
```

Comment:
You consider "Hobbies" as a subsection of "About me", right? Otherwise, it should be an h2 as well.

Reply:
Correct, it belongs to the "About me" section